### ISSUE TITLE
Ability to set custom spacing for grids etc

### ISSUE THREAD

**Issue by Atque** [Stellarium/stellarium#1262]
Currently, we're stuck to one set of spacing for grids, which makes them very "noisy", see the screenshot below:

![stellarium-026](https://user-images.githubusercontent.com/37028368/92993218-1af7d780-f4f0-11ea-9e0c-8fbe2e05c0ec.png)

It would be great to have an option to have the grids more "coarse" or "fine". Starry Night Pro Plus has this feature, and Cartes du Ciel has a similar feature.

Also, a similar request: I would like the partitions on the ecliptic etc to get finer once zoomed in. As of current, the partitions and lables are fixed and won't change when zoomed in. See this screenshot:
![stellarium-027](https://user-images.githubusercontent.com/37028368/92993270-6f02bc00-f4f0-11ea-97d8-49dd837bc5dc.png)

**Comment by gzotti** [comment]
So you want to disable/override automatic densification for grids, and enable some automatic or manual densification for the partitions. How fine do you think they should become?

**Comment by Atque** [comment]
@gzotti No, not override, but make it more coarse or fine depending on what the user wishes for. About the partitions, I think they could be just as fine as the grid markings.

**Comment by alex-w** [comment]
@gzotti: @Atque said about Cartes du Ciel in the part 1 and apparently he want to get the tool to custom definition for values of STEP_SIZES_DMS and STEP_SIZES_HMS arrays (GridLinesMgr.cpp:195-196)
